Source/Neptune.Web/Views/ManagerDashboard/Index.cshtml

```diff
@@ -0,0 +1,62 @@
+﻿@*-----------------------------------------------------------------------
+    <copyright file="Index.cshtml" company="Tahoe Regional Planning Agency">
+    Copyright (c) Tahoe Regional Planning Agency. All rights reserved.
+    <author>Sitka Technology Group</author>
+    </copyright>
+
+    <license>
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Affero General Public License <http://www.gnu.org/licenses/> for more details.
+
+    Source code is available upon request via <support@sitkatech.com>.
+    </license>
+    -----------------------------------------------------------------------*@
+@using Neptune.Web.Views
+@using Neptune.Web.Views.Shared
+@using LtInfo.Common.DhtmlWrappers
+@inherits Neptune.Web.Views.ManagerDashboard.Index
+@section JavascriptAndStylesContent
+{
+    @{ DhtmlxGridIncludes.RenderPartialView(Html); }
+
+}
+
+<div class="pageIntroText">
```

Comment:
What does the class pageIntroText do? Should it be moved into the ViewPageContent partial view?

Reply:
Since pageIntroText isn't defined anywhere, and removing the class does not affect the look of the page, I removed the class everywhere it is being used (7 places).